exercism/largest-series-product/largest-series-product.js

```
@@ -0,0 +1,41 @@
+module.exports = class Series{
+  constructor(series){
+    this.series = series,
+    this.container = [],
+    this.biggestProduct = 0
+  }
+
+  largestProduct(largestOne){
+    console.log('here', this.series, largestOne);
+    if(!this.series && largestOne > 0) {
+      throw new Error('Slice size is too big.')
```

A few things here: 
1) This if block and the else if on line 16 throw the same error, and could be rewritten to something like:
```
if((!this.series && largestOne > 0) || (largestOne > this.series.length)){
      throw new Error('Slice size is too big.')
}
```
2) But if this.series is null or undefined, that should really throw a different error - the problem in that case isn't that the slice size is too big, it's that the series constructor was passed invalid input, which should be checked and handled when the invalid input is receive. So you'd only need the largestOne > this.series.length part.

See above.